src/components/PostCard/PostHeader/__snapshots__/PostHeader.test.js.snap

```diff
@@ -163,7 +163,9 @@ exports[`matches announcement snapshot 2`] = `
   </div>
   <div
     data-stylename="timeWindow"
-  />
+  >
+    request ended 
+  </div>
```

Comment:
These show up because I removed the possibility that end date is an empty string.